index.html

```diff
@@ -6,9 +6,307 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Homework 6</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css" integrity="sha512-NmLkDIU1C/C88wi324HBc+S2kLhi08PN5GDeUVVVC/BVt/9Izdsc9SVeVfA1UZbY3sHUlDSyRXhCzHfr6hmPPw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="./style/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Dosis&display=swap" rel="stylesheet">
 </head>
 <body>
+    <div class="wave">
+        <img src="./img/bg.png" alt="1">
+        </div>
+
+    <div class="header-start">
+        <div class="logo"><img src="./img/KIDS-jump-TECH-2 2.png" alt="jump"></div>
+        <div class="products">products</div>
+        <div class="test-drive">test drive</div>
+        <div class="for-kids">for kids</div>
+        <div class="games">games</div>
+        <div class="contacts-us">contacts us</div>
+        <div class="faq">faq</div>
+    </div>
+
+    <div class="top-content">
+
+    <div class="content-left">
+        <div class="game">
+            <img id="hide-icon" src="./img/Polygon 4.png" alt="2">
+            <img id="game-png" src="./img/Games.png" alt="1">
+        </div>
+        <div class="text">
+            More than 90 educational games and applications are included <br> with the delivery of the interactive floor.
+        </div>
+    </div>
+
+    <div class="content-right">
+        <div class="icon1">
+            <img id="white-icon1" src="./img/Polygon 24.png" alt="1">
+            <img id="icon-200" src="./img/200+.png" alt="11">
+        </div>
+        <div class="icon2">
+            <img id="white-icon2" src="./img/Polygon 24.png" alt="2">
+            <img id="icon-group" src="./img/Group.png" alt="22">
+        </div>
+        <div class="icon3">
+            <img id="white-icon3" src="./img/Polygon 24.png" alt="3">
+            <img id="icon-brain" src="./img/knowledge 1.png" alt="33">
+        </div>
+    </div>
+    
+    </div>
+
+    <div class="tag-menu">
+    <div class="tag-button">#adventure</div>
+    <div class="tag-button">#collaboration</div>
+    <div class="tag-button">#memory</div>
+    <div class="tag-button">#nature</div>
+    </div>
+
+<div class="main-cards">
+    <div class="Bath-Trouble">
```

Comment:
you don't need a separate class for every game

you can just add one class `game` for every game block